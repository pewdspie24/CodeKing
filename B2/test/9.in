88
11811310910010511111111610311710310711711611199
111108110120102104108109113119105101115
10311610410710110610610212011011510110110311610710112211512211810710111911911511411997111119105
1021001001009712010810998117114120
9712097122991171051201171221031151011061221161021201059911598117120118116115102115121111100121111109105117110109117116119112107106108117112122122119102105107106106105111107122
118111117101110119108101108116112114105101104108120117101
11812110111210812110312211210810210112011612211711111311911811997108105
100114115114121107113111121113107113122120991101149910210510510410112111998107120102111117110101117112103118109112117105111110122102110115103979911611011710210511112211011711210499991079810010211210310711011111611199122112106118105113981201039811210899119109
1151081021221041171001161091131031081221031151111001191041121071131201051141031031051211221091151221161001051159910510811210710510111811111597100110117105113111101102110117103118105122100100991181011121191121219912298116
9911711510410810011812011610410111811711310911610312010710497104121122121971201001221191081151051021041221041001119911211710210410312210611211611797115118108100117114122107121991071011141021151201051011041121111169811310911611811010110797100110117105121106118107103116104107110
122111103981221211101151161201121181091121141221031131211151191029912210510111311511210910711211910897115115107110119110111118118100113
11499107117979910311110010412211010710710610510311112211510310310011811399991131041179810610712211397114981101171151071169811612111410910210411210710911411510912110311211110410510510010111498971151001101011021011051199911798
12110310812110799989811911811010210311011510610410699117118107100103107112114991181179711997103103991089711512110910711310312011310610410211010310812010911799119119111114110112111117103121115112107120106120101104119122121115104
106107113105107107981171211031029712112112211010212212011810110811299979912111412110097107122108109106111106106114103111113114103118105
11711111411510910112012198122103112121103117115110121991071141031021091001101201099810610810510410711011010311411610010410010910311412299981181011051071191071051121071211011019899118116113
9711010610311710111812098111100109121108102110991091121031171221001069997981151211101081011201029711311211610011011098117105107119
107115105109971111181179711510110011999118981221011221201191031011209710511897113122106109114122105119113100106112101115114107114110999812110712010110810311210898113115100114101117119109122119110100122108121113100102121115106111113116100108115108103111101121120111104110102114110
971201061139912097101971051161041061079711111512211298105106101112110114104102110101119103117107122106102117104103991101071011221031151191061019810111912111311511211310611611511698119115116114122120115106121104116113
1229811211210812210310611711110310811099117110113108121114991011081141191061071211071089910511711810212011911912210310810610310211411911111910011810911197115115109117110105115112120114
107107118117119113109107112104111106121108119116100112991181061011071041219910597118108107118113102121118103102110103108114121118108102981011221111021199810010011910110510298119100107105
100999811311710310210911910911810710210510598104114121114101109122114100106108
1229711310910410211410810210612199979911211211411610810111411697971049911811811299115107102
12011610910010010211511097121108100108104110116117102120113112108122106111989910310211610711510510810612110311111998120116110122
1211141141201211021129711610810410010698117979710110512211997112113117118107113106115119121112101112104114106
10012210511912011811410098108
1101121041159812198111981011159811410610112110010510910410599122102122103103106110102120101106105115110971161041211171159810610410510511911610011110899119114113104
1161221001221041071191181181171011101129910010111299101110108118112112107118107105119103108989798989812110399102108111
11511599103101116115100105103103105110116110101108
11412110098104113100991151041061061061191181181179810997115103120113105122999912211510310012111411410012011810411411811810199112112104114109114104115115108100109122120102104118113109111118105112118115120113
118118118108107991221091051021141051001151189810610211211411499118113115991101121131221171071011111151121151129710910611997119103111119111111118106971141029711811597107116981071161091141041121011191051131021019997112107113121120109103
1181049711112012197108114112113113110121108105118108
1171151061211131171041111199999111121981151151051121141161051221111221061221011171151021071181089811810310510498112121110119121110107
114103117115116122999911210410311811411311398114120121107116981121041011079910810211912210210911310311011711111798100119113971111051121041069910310810712110312010111410199105102
11711111611711111097110111112102117115109111111119122107119100108
120120111102120116981211111071211221031051051021041051121111041191031111191121011101021099910097104981011091071131021111021161111221071149911711210710997122108110105106104
10911911711110811710810311111910311011112110610612011711710911811310610410099109117119981221121141081179899110100109110100101112116117111104116108122119110991101061071021081031179711410012011610811310697116116110102111
110971061191171111221131061179899108101111117114102122117120113116118116101107118104103112106117981229711912111811712011810611510610911997118118989810397100108100991139811912297106118991131001131211059910011811910010410710798110121113102105122971169799114113104
109107121117102991041111031001151001149710212111611911911010711012112211911910411811712211710099
12211012010111910411710910611711610610911599114120114991161141071091091209897108104120104112111121108101121108107117120108104117100114991171021131211171081011041081071201161161111069799100119
10610798116115115102103117100110111115115106110119112107111981101011171191221071051109999
11411310210710110010698110101107105120116108107991051119799110112115108122102100114115118
104981221181101069910010110612210112110111711110011812110010110111112010310811910110111011210797108112119107118111971141161221091161161031019911010111711512010911810099
112119981041151001151111111171111001021041039812010211911511111311598106104117100103103104101110119101117102101104122122110121105104120108111108119
121103111115111100971039899101120103116120109114102122115115105105117122114120111121114122105113971219811410011611410010610797112112100118111121119122116108971181159910410910797121119111111112115105117109119117109105110112107121114
110979812110510211311210412110810997119115105113113105119106105981091191111111131031081171041101121011051111131209911310710310710710012011510710011599102104999711010312010297106
109119119110119101102111113121116110112103117119120117117120113115109105110114105120122118114104981081021101221021121151201141101021221191101051131191089811410011710410111610211811310299100101111971021219710711910810099113115971001201071031029911211111010111012298114108105114118
9710710411211112110712210711110110210911110811711910511510411910611612110611211812010811397981211199711811198122117116106104104
107115102971151079911712010611810712012111110110111410498119113115112113
118112118121122122103106107121101114122103107112112119118109
1011101021091201181101229897113103102114121100112101112120116121108105104100111112100105106107120113116103110991141171151151179910311910798991081051081171229912111297105102113108119107104115
107991091141071071111151001061141169911510610711010810610212012199114103101121121103102117
100981189811910610011010911010310810410811610199118106999810511610911711810011711010612011098109119110102117104971141101071229798115102116105
112100104100106115111113101120106106103117112121104117107991061001051069911312010210397
1061161051061131109910210912211410710811111711012010197102112991191091121151111081159711199112107102103106971001051121201061191151101161021081051151031001221051201131061061041041191091011129710411810311299109108110121
108107110104109104118115119101113107106121981151061131201101119810810710910410710411411210410311912010199100119112106113971111021131029811399119106112107115112100118108114104114101102105112
1081001041041081151081161149710710411011410512011211111110210311411512112097104108106971221061101079799113107122118105109108110113122118116105
115100102113989811497114108119114112101103100115106102113121104971191071041211151101021199911610711297120
115999899981161071121171069711311597
9911610511911111012112012210310610111211412010711411010310811911899122114112102109108122108120104121
10412011011010411910399118104105108121119119101101121105981051051071201101171221051071221021011039911911499109112991009711699109113122114119108
1131211141031201181131001021061039910510497119119112109
106971031181109710998107118116102113104116115112105115971011141201061141001161169712097
11599100111108120104999812212099117105104101102117117991021161091171051131121071081011039711498104120122117120116114106113108114121114
115109120113113103104108119106119112120103111120117108981221079810310111911610910710999100
10311810811311012010011510311111897109111111114104103114110108106116112118119114103117981141071019999118
106116101102121111104121111102108119108114121107118106120112113104113117103113104118113
118107971201051091101191141081171189711310610410611312111810610110410610910710011411911710712011610310710211011511411410712010711211610397116120106102103115118102114103
1111141091011161091111101051001199999110108118104107118109111105981131181061181141071141181061071181171021181131069812011812111210811111711910511211112110312010712111511410510810910110510610410610811610410110110411997107106104119
109117113118989712011610312110911011111411210910410910397115120118114116121105106109100971039811310711811197103107102118118121109122115105114111981021051191051101101201121081211121151201101141161021099811312011798112100
1041221041051101201151161121151161099711010610710398116101104111116103100111104109111116110120100121981209912110910512299113
109117107116117105115104107107109102122122109117991179710410510211011011911810211710898116101109108991229911910611711410911511710511512099121107115108120991051151071221189710410197108111103115112109121108122102111105
989910511310510198113105120101971001181151029999122100117109101100116114116991081051041021021001181071211021071211001049810699117116111119102122104111105112
100105112108119114120100115115108120109122101122100112115113971071071171071001019997122971001151081149711997115114119114102119106104120101102120116103112101106105991161041041071111121091001221051201191179912199120
10711311010111011310110799102991161089998112107117114971151021109711011611011410311611712110010412211210310011511011110711710112210910511810210610711111912211910412110412110211610410412111510612111812011011510998
1179910310110111311810999107109109120118100118116101120100107103108120106120122999910211212211711398116991149911812112211499117122102118114115105101108121118117102112118121121106117105112117120100
1199912111099104115108114109105112981111071111001131129711210712011810910111610511412011511210010811099115122117999999122102122102114121991211201229911310411610511911811110211598116101101
1049911611611210312112110511897971151161171051209710610511410810110811812212111010710510010210910711710511110510211511012011710510210998117110122109115971019710711497105112106102122109100113115103115122
1101201001161051181151099811210111410311311712210210011011811311010799122111104110104116121119103113
10211610311110011010097104102111111113102121101118117111105121
10397121122101111113103104107100981221071191091221201041209910911611211711910510510011511410812111797104105109103991071081171061181161071141071101141161021141201001059912098122107102108110106114121110106110104117971101111111031029811412110811110911711910011110110699103104107
1019812111810411710210812210011710998122119111971121151201031221221061121171031061201071181169810099101113107118119119101100971021081181131101031129811910111712111811699105112121116116102102119114104117120119100122108102115121112115
1001081159911611910311011512210911799103120101112114103106108113102120113104106115117
118120104103105971111071211061011221171021161119811510510311612011897989910312010811410411810997101109102120109110106971161031031069912111610711111710411410910811211710612010711411511312111598117102101110
1191131011001101131009711211310311510310798115105111111104114109114120119111
11210610510510897118103101109122117103971051131151061109911097108105101103111971091101081161221121099911010610511510210210298106103113110100121111101107120122121117112107111101101102116
97116108106112102122120118118121117991011201071169711310899971001189710698118971141199812110711510311299116110105120117120971121091019911312211810110210810411211410397117118114971119812211411911011710011411312210810510310399105119112114105108108112100108106
991161221021081011151121061041161061071061121031061131089911010011312110610510111311911011112210410312297108100121105
1151079912099108119106991211161101221071181151021071161149910911710311312111298111119991191191061021201111171091201021131081019710712010999
