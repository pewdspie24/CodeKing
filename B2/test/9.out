vqmdiootgugkutoc
olnxfhlmqwies
gthkejjfxnseegtkezszvkewwsrwaowi
fdddaxlmburx
axazcuixuzgsejztfxicsbuxvtsfsyodyomiunmutwpkjlupzzwfikjjiokz
vouenwleltpriehlxue
vyeplygzplfextzuoqwvwali
drsrykqoyqkqzxcnrcfiiheywbkxfouneupgvmpuionzfnsgactnufioznuphcckbdfpgknotoczpjviqbxgbplcwm
slfzhudtmqglzgsodwhpkqxirggiyzmsztdiscilpkievosadnuiqoefnugvizddcvepwpyczbt
cushldvxthevuqmtgxkhahyzyaxdzwlsifhzhdocpufhgzjptuasvldurzkyckerfsxiehpotbqmtvnekadnuiyjvkgthkn
zogbzynstxpvmprzgqyswfczieqspmkpwlassknwnovvdq
rckuacgodhznkkjigozsggdvqccqhubjkzqarbnusktbtyrmfhpkmrsmygpohiiderbasdnefeiwcub
yglykcbbwvnfgnsjhjcuvkdgkprcvuawaggclasymkqgxqjhfnglxmucwwornpougyspkxjxehwzysh
jkqikkbuygfayyznfzxvelpcacyrydakzlmjojjrgoqrgvi
uorsmexybzgpygusnyckrgfmdnxmbjlihknngrtdhdmgrzcbveikwkipkyeebcvtq
anjguevxbodmylfncmpguzdjcabsynlexfaqptdnnbuikw
ksimaovuasedwcvbzezxwgexaivaqzjmrziwqdjpesrkrncbykxelgplbqsdreuwmzwndzlyqdfysjoqtdlslgoeyxohnfrn
axjqcxaeaithjkaoszpbijepnrhfnewgukzjfuhgcnkezgswjebewyqspqjtstbwstrzxsjyhtq
zbpplzgjuoglncunqlyrcelrwjkyklciuvfxwwzgljgfrwowdvmoassmunispxr
kkvuwqmkphojylwtdpcvjekhyciavlkvqfyvgfnglryvlfbezofwbddweifbwdki
dcbqugfmwmvkfiibhryremzrdjl
zaqmhfrlfjycacpprtlertaahcvvpcskf
xtmddfsnayldlhntufxqplzjobcgftksiljygowbxtnz
yrrxyfpatlhdjbuaaeizwapquvkqjswypephrj
dziwxvrdbl
nphsbybobesbrjeydimhiczfzggjnfxejisnathyusbjhiiwtdolcwrqh
tzdzhkwvvuenpcdepcenlvppkvkiwglbabbbygcflo
sscgetsdiggintnel
rydbhqdcshjjjwvvubmasgxqizcczsgdyrrdxvhrvvecpphrmrhssldmzxfhvqmovipvsxq
vvvlkczmifridsvbjfprrcvqscnpqzukeospspamjwawgowoovjarfavsaktbktmrhpewiqfecapkqyxmg
vhaoxyalrpqqnylivl
usjyquhowccoybssiprtizozjzeusfkvlbvgihbpynwynk
rgustzccphgvrqqbrxyktbphekclfwzfmqgnuoubdwqaoiphjcglkygxerecif
uotuonanopfusmoowzkwdl
xxofxtbyokyzgiifhipohwgowpenfmcdahbemkqfoftozkrcupkmazlnijh
mwuolulgowgnoyjjxuumvqjhdcmuwbzprlubcndmndeptuohtlzwncnjkflguardxtlqjattnfo
najwuozqjubcleourfzuxqtvtekvhgpjubzawyvuxvjsjmwavvbbgadldcqbwzajvcqdqyicdvwdhkkbnyqfizatacrqh
mkyufchogdsdrafytwwnknyzwwhvuzudc
znxewhumjutjmscrxrctrkmmxbalhxhpoyleylkuxlhudrcufqyulehlkxttojacdw
jkbtssfgudnossjnwpkobneuwzkincc
rqfkedjbnekixtlkcioacnpslzfdrsv
hbzvnjcdejzeyeuodvydeeoxglweenpkalpwkvoartzmttgecneusxmvdc
pwbhsdsoouodfhgbxfwsoqsbjhudgghenweufehzznyihxlolw
ygosodagbcexgtxmrfzssiiuzrxoyrziqaybrdtrdjkappdvoywztlavschmkaywoopsiumwuminpkyr
nabyifqphylmawsiqqiwjibmwooqgluhnpeioqxcqkgkkdxskdscfhcangxfaj
mwwnwefoqytnpguwxuuxqsminrixzvrhblfnzfpsxrnfzwniqwlbrduhetfvqfcdeoafyakwldcqsadxkgfcponenzbrlirv
akhpoykzkoefmoluwishwjtyjpvxlqabywavobzutjhh
ksfaskcuxjvkxyoeerhbwqspq
vpvyzzgjkyerzgkppwvm
enfmxvnzbaqgfrydpepxtylihdopdijkxqtgncrussucgwkbcliluzcypaifqlwkhs
kcmrkkosdjrtcsjknljfxycrgeyygfu
dbvbwjdnmnglhltecvjcbitmuvdunjxnbmwnfuharnkzabsfti
pdhdjsoqexjjgupyhukcjdijcqxfga
jtijqncfmzrklounxeafpcwmpsolsaocpkfgjadipxjwsntflisgdzixqjjhhwmepahvgpcmlny
lknhmhvsweqkjybsjqxnoblkmhkhrphgwxecdwpjqaofqfbqcwjpkspdvlrhrefip
ldhhlsltrakhnrixpoofgrsyxahljazjnkacqkzvimlnqzvti
sdfqbbrarlwrpegdsjfqyhawkhysnfwctkpax
scbcbtkpujaqsa
ctiwonyxzgjeprxkrnglwvczrpfmlzlxhy
hxnnhwgcvhilywweeyibiikxnuzikzfegcwrcmpcdatcmqzrwl
qyrgxvqdfjgcihawwpm
jagvnambkvtfqhtspisaerxjrdttaxa
scdolxhcbzxcuihefuucftmuiqpklegarbhxzuxtrjqlryr
smxqqghlwjwpxgoxulbzkbgewtmkmcd
gvlqnxdsgovamoorhgrnljtpvwrgubrkeccv
jtefyohyoflwlrykvjxpqhqugqhvq
vkaximnwrluvaqjhjqyvjehjmkdrwukxtgkfnsrrkxkptgatxjfgsvfrg
ormetmonidwccnlvhkvmoibqvjvrkrvjkvufvqjbxvyplouwipoygxkysrilmeijhjltheehwakjhw
muqvbaxtgymnorpmhmgasxvrtyijmdagbqkvoagkfvvymzsirobfiwinnxplypsxnrtfmbqxubpd
hzhinxstpstmanjkgbtehotgdohmotnxdybxcymizcq
muktuishkkmfzzmucuahifnnwvfulbtemlczcwjurmsuisxcykslxciskzvahealogspmylzfoi
bciqiebqixeadvsfcczdumedtrtclihffdvkyfkydhbjcutowfzhoip
diplwrxdsslxmzezdpsqakkukdecazadslrawasrwrfwjhxefxtgpejicthhkopmdzixwucycx
kqnenqekcfctlcbpkurasfnantnrgtuydhzpgdsnokuezmivfjkowzwhyhyfthhysjyvxnsmb
ucgeeqvmckmmxvdvtexdkglxjxzccfpzuqbtcrcvyzrcuzfvrsielyvufpvyyjuipuxd
wcynchslrmipbokodqpapkxvmetirxspdlncszuccczfzfrycyxzcqhtiwvofsbtee
hcttpgyyivaastuixajirlelvzynkidfmkuioifsnxuifmbunzmsaeakraipjfzmdqsgsz
nxdtivsmbpergquzfdnvqnkczohnhtywgq
ftgodndahfooqfyevuoiy
gayzeoqghkdbzkwmzxhxcmtpuwiidsrlyuahimgcklujvtkrknrtfrxdicxbzkflnjrynjnhuanoogfbrylomuwdoejcghk
ebyvhuflzdumbzwoapsxgzzjpugjxkvtbdceqkvwwedaflvqngpbweuyvtcipyttffwrhuxwdzlfsyps
dlsctwgnszmucgxeprgjlqfxqhjsu
vxhgiaokyjezuftobsigtxvabcgxlrhvmaemfxmnjatggjcytkouhrmlpujxkrsqysbufen
wqednqdapqgsgkbsioohrmrxwo
pjiilavgemzugaiqsjncnaliegoamnltzpmcnjisfffbjgqndyoekxzyupkoeeft
atljpfzxvvyucexktaqlcadvajbvarwbyksgpctnixuxapmecqzveflhprgauvraobzrwnudrqzliggciwprillpdlj
ctzflespjhtjkjpgjqlcndqyjieqwnozhgzaldyi
skcxclwjcytnzkvsfktrcmugqypbowcwwjfxoumxfqleakxmc
